xwiki-platform-core/xwiki-platform-release/xwiki-platform-release-test/xwiki-platform-release-test-docker/src/test/it/org/xwiki/release/test/ui/ReleaseIT.java

```diff
@@ -76,9 +85,9 @@ public void testRelease()
         homePage.waitUntilPageIsLoaded();
 
         // Assert Livetable:
-        // - verify that the Translation has been applied by checking the Translated livetable column name
-        // - verify that the Livetable contains our new Release entry
-        LiveTableElement lt = homePage.getReleaseLiveTable();
-        Assert.assertTrue(lt.hasRow("Version", RELEASE_VERSION));
+        // - verify that the Livetable contains our new Release entry.
+        LiveDataElement liveDataElement = homePage.getReleaseLiveData();
+        assertTrue(liveDataElement.hasRow("Version", RELEASE_VERSION),
+            String.format("%s expected to be found in column Version.", RELEASE_VERSION));
```

Comment:
Straightforward migration of the existing test, more assertions are needed here.